test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AccountBalancesClientSaveLoadTest.java

```diff
@@ -200,18 +212,23 @@ private HapiApiSpec runAccountBalancesClientSaveLoadTest() {
 			@Override
 			public Optional<HapiSpecOperation> get() {
 				int next;
-				if ((next = createdSofar.getAndIncrement()) >= totalAccounts) {
+				next = moreToCreate.getAndDecrement();
```

Comment:
nit: should be in a single line as in my original comment.

Reply:
changed.